frontend/class-opengraph.php

```diff
@@ -576,7 +576,14 @@ public function image( $image = false ) {
 		$opengraph_images = new WPSEO_OpenGraph_Image( $this->options, $image );
 
 		foreach ( $opengraph_images->get_images() as $img ) {
-			$this->og_tag( 'og:image', esc_url( $img ) );
+			$og_image_tag = 'og:image';
+
+			if (0 === strpos($img, 'https://'))
```

Comment:
Thanks for the explanation on how to run the code style linter locally.

I looked for docs about how to run the test suite locally but couldn't find anything.